keyboards/fancytech/fancyalice66/keymaps/via/keymap.c

```diff
@@ -0,0 +1,59 @@
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    /*
+     *    ┌───┬───┬───┬───┬───┬───┬───┐   ┌───┬───┬───┬───┬───┬───┬───────┐  ┌───┐
+     *    │Esc│ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │   │ 7 │ 8 │ 9 │ 0 │ - │ = │Backsp │  │Del│
+     *   ┌┴───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┘ ┌─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┴┐ └┬──┴┐
+     *   │ Tab │ Q │ W │ E │ R │ T │    │ Y │ U │ I │ O │ P │ [ │ ] │  \   │  │PgD│
+     *  ┌┴─────┼───┼───┼───┼───┼───┤    └┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴──────┴┐ └┬──┴┐
+     *  │ Caps │ A │ S │ D │ F │ G │     │ H │ J │ K │ L │ ; │ ' │  Enter   │  │PgU│
+     * ┌┴──────┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    └─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───────┬─┬┴──┼───┘
+     * │ Shift  │ Z │ X │ C │ V │ B │      │ N │ M │ , │ . │ / │  Shift  │ │Up │
+     * ├─────┬──┴───┼───┴─┬─┴───┴─┬─┴─┐  ┌─┴───┴───┴┬──┴──┬┴───┴───────┬─┴─┼───┼───┐
+     * │ Ctl │      │ Alt │       │FN │  │          │ Alt │            │Lft│Dwn│Rgt│
+     * └─────┘      └─────┴───────┴───┘  └──────────┴─────┘            └───┴───┴───┘
+     */
+
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_DEL,
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,             KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_PGUP,
+        KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,             KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,           KC_PGDN,
+        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,             KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,
+        KC_LCTL,          KC_LALT, KC_SPC,                    MO(1),   KC_SPC,                    KC_RALT,                   KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+
+    [1] = LAYOUT(
+        KC_TILD, KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  QK_BOOT,          KC_PSCR,
+        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, RGB_SAI, RGB_SAD, RGB_TOG, RGB_HUI,
+        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______, _______,          RGB_HUD,
+        _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,          RGB_VAI,
+	    _______,          _______, _______,                   _______, _______,                   RGB_MOD,                   _______, RGB_VAD, _______
+    )
+};
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [1] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS) },
+    [2] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS) },
+    [3] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS) },
```

Comment:
Has to match number of layers
```suggestion
    [1] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS) }
```